### PR TITLE
Display improvement in list command

### LINKED ISSUE #39: Display improvement in list command
Display improvement in list command

### PR DESCRIPTION
**Pull request by kamontia**
@chaspy Please review and merge

Close #39

### CODE REVIEW

File: kimetaro.py [hunk 1]
```diff
@@ -17,6 +18,7 @@ def __init__(self):
 
 # Global definition(Workaround)
 kimetaro = Kimetaro()
+_loop = asyncio.get_event_loop()
```
Comment (chaspy) [inline kimetaro.py]: What does `_` mean?

Comment (kamontia) [reply kimetaro.py]: @chaspy Fixed up.

**Comment by kamontia** [comment]
@chaspy Please review! If there no problem, please merge.

**Review by chaspy — APPROVED** [review]
LGTM 👍

**Pull request merged**